assignments/colorpicker.md

```diff
@@ -89,18 +85,16 @@ The height of the dial is the height of `ColorPickerView`, this is not true for
 
 ### Key Variables
 
-To do some parts of your drawing you will need to know the size of the
-view you are drawing in. Calculate `mRadius`, `mCenterX`, and
-`mCenterY` of the color picker inside `onLayout`
+To do some parts of your drawing you will need to know the size of the view you are drawing in. Calculate `mRadius`, `mCenterX`, and `mCenterY` of the color picker inside `onLayout`. 
 
 _Related APIs_:
 [View#onLayout](<https://developer.android.com/reference/android/view/View.html#onLayout(boolean,%20int,%20int,%20int,%20int)>)
 
 ### Thumb
 
-In the screenshots there is a visible thumb that marks the selected color on the dial. The thumb is drawn in `onDraw()` and should move around as a user interacts with the color picker, not just jumping to its final location when they release the finger/mouse.
+In the screenshots there is a visible thumb that marks the selected color on the dial. The thumb is drawn in `onDraw()` and should move around as a user interacts with the color picker.
 
-The thumb should be constrained to moving along a circular track that places it within the dial. It should move along that track even when responding to valid events within the wheel but not within the dial.
+The thumb should be constrained to moving along a circular track that places it within the dial. It should move along that track even when the user is interacting with the inner circle.
```

Comment:
"interacting with" isn't super clear in this case, maybe "dragging the pointer".